Library/TestHelpers/MockStripeIntentService.swift

```diff
@@ -0,0 +1,51 @@
+@testable import KsApi
+@testable import Library
+import ReactiveSwift
+
+public struct StripeIntentRequestTestData: Equatable {
+  let projectId: String?
+  let pledgeTotal: Double?
+  let paymentIntentEnvelope: PaymentIntentEnvelope?
+  let context: GraphAPI.StripeIntentContextTypes?
+  let clientSeretEnvelope: ClientSecretEnvelope?
+}
```

Comment:
One data structure to compare felt cleaner than having one for payment intents and one for setup intents. Also this way we can guarantee that only one intent request type (payment or setup) is being made at a time.